### PR TITLE
[Consensus] Add unit tests for failing empty signature verification

### PR DESCRIPTION
**Pull request by sitalkedia**
### Description

This is a follow up from previous PR.

### Test Plan

Unit test

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2754)

### CODE REVIEW

File: types/src/validator_verifier.rs [hunk 1]
```diff
@@ -556,6 +556,22 @@ mod tests {
...
+    #[test]
+    fn test_empty_multi_signatures() {
+        let validator_signer = ValidatorSigner::random(TEST_SEED);
+        let dummy_struct = TestAptosCrypto("Hello, World".to_string());
+        let validator =
+            ValidatorVerifier::new_single(validator_signer.author(), validator_signer.public_key());
+
+        assert_eq!(
+            validator.verify_multi_signatures(&dummy_struct, &MultiSignature::empty()),
+            Err(VerifyError::TooLittleVotingPower {
```
Comment (zekun000) [inline types/src/validator_verifier.rs:568]: isn't it EmptySignature error expected? can we fill in the bitmask?

Comment (sitalkedia) [reply types/src/validator_verifier.rs:586]: This behavior is expected, added another test for empty signature.

**Comment by sitalkedia** [comment]
Can I get a stamp on this @zekun000 ?

**Review by zekun000 — APPROVED** [review]
thanks for the followup!
